sktime/forecasting/model_evaluation/_functions.py

```diff
@@ -62,17 +63,44 @@ def evaluate(
 
     Examples
     --------
+        The type of evaluation that is done by `evaluate` depends on metrics in
+        param `scoring`
+        When evaluating model/estimators on point forecast, users can either let
```

Comment:
remove "either"; since there is not an "or"

Reply:
Done